src/enzo/Grid_RemoveMassFromGrid.C

```diff
@@ -0,0 +1,469 @@
+/***********************************************************************
+/
+/  GRID CLASS 
+/  Remove Mass from grid based on accretion rate calculated. 
+/
+/  written by: John Regan
+/  date:       July 2016
+/  modified1:
+/
+/  PURPOSE:
+/
+/  RETURNS:
+/
+************************************************************************/
+ 
+#include "preincludes.h"
+#include "ErrorExceptions.h"
+#include "macros_and_parameters.h"
+#include "typedefs.h"
+#include "global_data.h"
+#include "Fluxes.h"
+#include "GridList.h"
+#include "ExternalBoundary.h"
+#include "fortran.def"
+#include "Grid.h"
+#include "phys_constants.h"
+#include "ActiveParticle_SmartStar.h"
+#define TINY_NUMBER         1e-20
+#define SMALL_NUMBER         1e-6
+#define ACCRETION_LIMIT     9e-1
+#define N 8
```

Comment:
Should probably be undefined at the end of this file (or moved to macros_and_parameters if they're meant to be global).